### PR TITLE
Update: French doc

### PR DESCRIPTION
**Pull request by GitGud31**
Added 'all providers' section to the french documentation

### CODE REVIEW

**Comment by GitGud31** [comment]
Fixed: 
* Some broken links.
* Provider use case example code (English and French)

File: website/i18n/fr/docusaurus-plugin-content-docs/current/providers/provider/completed_todos.dart [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import 'todo.dart';
+
+/* DÉBUT DU SNIPPET */
```
Comment (rrousselGit) [inline website/i18n/fr/docusaurus-plugin-content-docs/current/providers/provider/completed_todos.dart:5]: These shouldn't be translated

They are keys used to remove the code before/outside of the comments

Comment (GitGud31) [reply website/i18n/fr/docusaurus-plugin-content-docs/current/providers/provider/completed_todos.dart]: Ah my bad, I reverted all of them back to English

Notice the build fail (error logs not related to files changed) not quite sure why, and how can I fix it?

Comment (GitGud31) [reply website/i18n/fr/docusaurus-plugin-content-docs/current/providers/provider/completed_todos.dart]: Ah my bad, I reverted all of them back to English

**Comment by rrousselGit** [comment]
Thanks! 😄

**Pull request merged**